web/owlspa/src/actions/serverSavePatch.js

```diff
@@ -63,16 +65,16 @@ const serverSavePatch = (patch, options = {}) => {
             } else {
               throw new Error('Error saving patch');
             }
-          } 
+          }
           return json;
         });
       })
       .then( json => {
-        
+
         if(json._id){
           dispatch({type: PATCH_SAVED});
           if(method === 'POST'){
-            dispatch(cleanUpTmpPatchFiles(json._id)).then(() => {            
+            dispatch(cleanUpTmpPatchFiles(json._id)).then(() => {
```

Comment:
Agreed, that's my own mess from 2 years ago :smile: Let's raise a separate issue for that.